### ISSUE TITLE
Vertical scrolling with lag and stutter

### ISSUE THREAD

**Issue by Focom** [Microsoft/vscode#35650]
- VSCode Version: 1.17
- OS Version: Win10
- Hardware : Dell XPS 9530

Steps to Reproduce:
1. Update from 1.16.1 to 1.17 
2. Use two finger vertical scroll anywhere
3. Experience lag and stutter

Reproduces without extensions: Yes

Vertical scrolling was working butter smooth on the older build, if there is no fix, can I downgrade to 1.16.1 ?

**Comment by Focom** [comment]
Seems to work now, i am closing the issue

**Comment by coopersamuel** [comment]
I've noticed this issue on previous versions of VS Code, but it seems to be worse on 1.17... Usually, on previous versions, I was able to work around the issue (recorded [here](https://github.com/Microsoft/vscode/wiki/Known-issues)). Now the usual workaround seems like it only fixes the issue temporarily. 

Steps to reproduce:
- Open VS Code v1.17
- Observe laggy touchpad scrolling
- Restore and re-maximize the window, this should alleviate the laggy scrolling for as long as you are in this window.
- Minimize the window, or navigate to a different window
- Reopen the VS Code window and observe that scroll lag has returned
